packages/core/src/shared/sam/constants.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable header/header */
+/* eslint-disable @typescript-eslint/naming-convention */
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const BUILD_PROCESS_MEMENTO_ROOT_KEY = 'samcli.build.processes'
```

Comment:
I think the standard we have in the toolkit is to use camal case, even for constants